store/dataStore.ts

```diff
@@ -30,4 +65,26 @@ export const useDataStore = create<DataStore>((set) => ({
       }),
     ),
   resetUser: () => set({ user: initialState.user }),
+  updateNFLTeams: (teamName, teamLogo) =>
```

Comment:
m: Alphabetize the function names.

Reply:
Reordered the functions in alphabetical order.